src/devices/Bno055/samples/Program.cs

```diff
@@ -9,7 +9,7 @@
 
 Console.WriteLine("Hello BNO055!");
 using I2cDevice i2cDevice = I2cDevice.Create(new I2cConnectionSettings(1, Bno055Sensor.DefaultI2cAddress));
-using Bno055Sensor bno055Sensor = new Bno055Sensor(i2cDevice);
+using Bno055Sensor bno055Sensor = new (i2cDevice);
```

Comment:
I'm personally not a big fan in this situation, `using var` seems a lot easier to parse to me. But I guess that's a style choice.
There's also not a lot of consistency in the pattern that's used in this, PR, some are using explicit types, some are using var, and this one is using target-typed new.

Reply:
I used to be in the `var` camp. I switched as part of the last couple PRs I've done on this codebase. I've never spent so much time in a codebase in which I was totally unfamiliar. It's not a problem for construction, which this specific comment is about, but is a problem for readability for everything else. I've decided I'm not going to use `var` any more.

Reply:
You are right that the PR isn't consistent on on this aspect. I'm trying some things. I still haven't figured out the right pattern. Fortunately, find and replace makes it somewhat straightforward to make changes.

Reply:
> I've decided I'm not going to use var any more.
😲🤯